docs/source/best_practices/table_extraction_pdf.rst

```diff
@@ -0,0 +1,72 @@
+Table Extraction from PDF
+=========================
+
+This section describes two methods for extracting tables from PDF files.
+
+.. note::
+
+    To extract tables from any documents, set the ``strategy`` parameter to ``hi_res`` in both methods.
+
+Method 1: Using `partition_pdf`
+-------------------------------
+
+To extract the tables from PDF files using the `partition_pdf <https://unstructured-io.github.io/unstructured/core/partition.html#partition-pdf>`__, set the ``infer_table_structure`` parameter to ``True`` and ``strategy`` parameter to ``hi_res``.
+
+**Usage**
+
+.. code-block:: python
+
+    from unstructured.partition.pdf import partition_pdf
+
+    filename = "example-docs/layout-parser-paper.pdf"
+
+    elements = partition_pdf(filename=filename
+                             infer_table_structure=True,
+                             strategy='hi_res',
+                            )
+
+    tables = [el for el in elements if el.category == "Table"]
+
+    print(tables[0].text)
+    print(tables[0].metadata.text_as_html)
+
+Method 2: Using Auto Partition or Unstructured API
+--------------------------------------------------
+
+For extracting tables from PDFs using `auto partition <https://unstructured-io.github.io/unstructured/core/partition.html#partition>`__ or `Unstructured API parameters <https://unstructured-io.github.io/unstructured/apis/api_parameters.html>`__ , three parameters are relevant: ``strategy``, ``skip_infer_table_types``, and ``pdf_infer_table_structure``.
+
+.. warning::
+    **Avoiding Conflicts**: A conflict arises if ``pdf`` is included in the list of ``skip_infer_table_types`` while ``pdf_infer_table_structure`` is set to ``True``. You will get a warning when these two parameters are in conflict.
+
+**Usage: Auto Partition**
+
+.. code-block:: python
+
+    from unstructured.partition.auto import partition
+
+    filename = "example-docs/layout-parser-paper.pdf"
+
+    elements = partition(filename=filename,
+                         skip_infer_table_types=[],
+                         pdf_infer_table_structure=True,
```

Comment:
Do you need `skip_infer_table_types` and `pdf_infer_table_structured`? I thought `skip_infer_table_types` superseded `pdf_infer_table_types`.

Reply:
It seems both params must be defined when using `auto` partition to extract tables. This is because the `skip_infer_table_types` includes `pdf` by default. If the `pdf_infer_table_structured` is enabled, without removing `pdf` in `skip_infer_table_types`, it'll trigger a warning.
Ref: https://github.com/Unstructured-IO/unstructured/blob/main/unstructured/partition/auto.py#L564

Reply:
Clarification from @yuming-long 
- `skip_infer_table_types` won't supersede `pdf_infer_table_structure` (otherwise it is a bug)
- user should set `skip_infer_table_types` to a list without "pdf" for example ["jpg"] or []
- if the "pdf" in `skip_infer_table_types` is removed, and the user doesn't want to do table extraction for pdf, then do not set `pdf_infer_table_structure` to "True"; Otherwise, we will do table extraction since we still use both parameters to decide table extraction, so: (i)if the "pdf" in `skip_infer_table_types` is removed, `pdf_infer_table_structure` to "True". do table extraction, give a warning. (ii) if the "pdf" in `skip_infer_table_types` is removed, `pdf_infer_table_structure` to "False", no table extraction (and of no warning)
There is no need to define both, but just if there is a conflict between the two, give a warning.

Reply:
We should have both of those parameters in the docs, that'll get confusing. Keep `skip_infer_table_types` and drop references to `pdf_infer_table_types`.

Reply:
@ron-unstructured - I'll create a follow on issue to deprecate one of those parameters. As far as this docs PR goes, we're good to go here.

Reply:
i tested this and got table output
```
from unstructured.partition.auto import partition

elements = partition("layout-parser-paper.pdf", strategy='hi_res',
    skip_infer_table_types=[],
)

print([el.metadata.text_as_html for el in elements if el.metadata.text_as_html])
```
i think only set `skip_infer_table_types` should do the trick for pdf